fixtures/ios_app_with_remote_binary_swift_package/Project.swift

```diff
@@ -0,0 +1,29 @@
+import ProjectDescription
+
+
+let project = Project(name: "App",
+                      packages: [
+                        .package(url: "https://github.com/firebase/firebase-ios-sdk", .upToNextMajor(from: "7.1.0")),
+                      ],
+                      targets: [
+                        Target(name: "App",
+                               platform: .iOS,
+                               product: .app,
+                               bundleId: "io.tuist.App",
+                               infoPlist: "Support/Info.plist",
+                               sources: ["Sources/**"],
+                               resources: [],
+                               dependencies: [
+                                    // Firebase analytics is a binary target
+                                    .package(product: "FirebaseAnalytics"),
```

Comment:
If anyone knows of a smaller binary dependency we can use for this fixture that would be welcome :)

Reply:
What about a local package to prevent flakiness that might come from the network connection? Would that cover what you are trying to fix here?

Reply:
Good idea I was able to reproduce using a local Swift Package with binary dependency - more prebuilt .xcframeworks 🤐